src/index.js

```diff
@@ -3,6 +3,7 @@ var HyperResponse = require( "./hyperResponse.js" );
 var HyperResource = require( "./hyperResource.js" );
 var jsonEngine = require( "./jsonEngine.js" );
 var halEngine = require( "./halEngine.js" );
+var HttpEnvelope = require( "autohost/src/http/httpEnvelope" )();
```

Comment:
@arobson Is this a new dependency? It seems like you probably don't want a dependency on autohost, but if we do have one then we need to add it to the deps in the package.json.